service.librarywatchdog/core/main.py

```diff
@@ -180,7 +180,7 @@ def main():
         start = time.time()
         while time.time() - start < settings.STARTUP_DELAY:
             xbmc.sleep(100)
-            if xbmc.abortRequested:
+            if xbmc.Monitor().abortRequested():
```

Comment:
if this is a long running task it is better to define a monitor before the while clause and reuse that instance (otherwise you're creating a new xbmc.Monitor() every iteration.

Reply:
Done